src/components/Modals/DeployModal/DeployToWorld_WorldsForEnsOwnersFeature/DeployToWorld.tsx

```diff
@@ -0,0 +1,456 @@
+
+    return `https://twitter.com/intent/tweet?text=${text}&url=${url}`
+  }
+
+  const renderEmptyState = () => {
+    return (
+      <div className={styles.emptyState}>
+        <div className={styles.modalHeader}>
+          <h3>{t('deployment_modal.deploy_world.empty_state_title')}</h3>
+        </div>
+        <div className={`${styles.modalBodyState} ${styles.modalBodyEmptyState}`}>
+          <div className={styles.emptyThumbnail} aria-label={project?.description} role="img" />
+          <span className={styles.description}>
+            {t('deployment_modal.deploy_world.empty_state_description', {
+              br: () => <br />,
+              b: (text: string) => <b>{text}</b>
+            })}
+          </span>
+        </div>
+        <div className={styles.modalBodyStateActions}>
+          <Button primary className={styles.modalBodyStateActionButton} onClick={handleClaimName}>
+            {t('deployment_modal.deploy_world.claim_name')}
+          </Button>
+          <Button
+            secondary
+            className={styles.modalBodyStateActionButton}
+            as="a"
+            href="https://decentraland.org/blog/project-updates/manage-names-in-the-builder"
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            {t('global.learn_more')}
+          </Button>
+        </div>
+      </div>
+    )
+  }
+
+  const renderSuccessState = () => {
+    return (
+      <>
+        <div className={`${styles.modalBodyState} ${styles.modalBodySuccessState}`}>
+          <div className={styles.successImage} aria-label={project?.description} role="img" />
+          <h1 className={styles.modalHeader}>{t('deployment_modal.deploy_world.success.title')}</h1>
+          <span className={styles.description}>{t('deployment_modal.deploy_world.success.subtitle')}</span>
+          <CopyToClipboard role="button" text={getExplorerUrl} className={`${styles.shareUrlField} field`} showPopup={true}>
+            <Field
+              className={styles.shareUrlFieldInput}
+              value={getExplorerUrl}
+              kind="full"
+              readOnly
+              icon={<DCLIcon aria-label="Copy World" aria-hidden="false" className="link copy" name="copy outline" />}
+            />
+          </CopyToClipboard>
+        </div>
+        <div className={styles.modalBodyStateActions}>
+          <Button
+            primary
+            className={styles.modalBodyStateActionButton}
+            onClick={handleNavigateToExplorer}
+            content={t('deployment_modal.deploy_world.success.jump_in')}
+            icon="external alternate"
+          />
+          <Button
+            as="a"
+            secondary
+            className={styles.modalBodyStateActionButton}
+            content={t('deployment_modal.deploy_world.success.share_in_twitter')}
+            icon="twitter"
+            href={getShareInTwitterUrl()}
```

Comment:
We could track the twitter sharing action if we would like to, what do you think about it?

Reply:
It might be a good idea to track it.

https://github.com/decentraland/builder/issues/2921 Created an issue in the epic to do so